packages/web-components/src/components/content-item-horizontal/__stories__/content-item-horizontal.stories.ts

```diff
@@ -63,6 +64,34 @@ export const Default = ({ parameters }) => {
   `;
 };
 
+export const WithThumbnail = ({ parameters }) => {
+  const { alt, heading, copy, ctaType1, ctaCopy1, href1, ctaType2, ctaCopy2, href2 } =
+    parameters?.props?.ContentItemHorizontal ?? {};
+  return html`
+    <dds-content-item-horizontal thumbnail="true">
```

Comment:
Where is this `thumbnail` attribute being defined? I'm not seeing this prop defined in the `dds-content-item-horizontal` component.